### ISSUE TITLE
Help offered for next release

### ISSUE THREAD

**Issue by paul-rouse** [purescript-halogen/purescript-halogen#633]
@garyb I was wondering if there is anything I could do to help make progress on Halogen 6.

I have two Halogen projects going at the moment, both of which are complex enough to exercise all of the changes relative to v5.  I've tried patched versions of them against the next-6 branch, and I like the result, especially for simple cases of event sources!  Don't get me wrong: I have been hooked on Halogen ever since first using v0.12 (for a predecessor of one of the present projects), and v5 already made things pretty easy and clear; next-6 just continues to improve it!

Let me know what would help - reviewing documentation? testing? any changes to code?

In any case, thanks for a brilliant piece of software!

**Comment by garyb** [comment]
Thanks very much for the offer, and apologies it took a while to get back to you!

I finally have some time to work on Halogen bits again, so right now I'm just doing a comb through of the revised docs for v5, and once that is done I'll finally make a non-RC release for Halogen 5, which will probably also be the last version of v5 :smile:.

I don't really have any intent to change anything drastically in v6 other than what is already done. @thomashoneyman has expressed interest in introducing "portals", similar to the feature in React, but I don't think that will result in any changes to the existing API.

For now, any feedback you have from using v6 is probably the most useful thing, or any suggestions you might have for further friction reduction. 

I'll have some completely new docs to read through soonish too, so review/suggestion for those would be most useful when I have something to show. It might seem crazy to throw out the current docs right after updating them, but I'd like to try and make them a reference, rather than trying to provide them as a guide. Jordan and Thomas have done a great job with providing additional documentation material that I'll be linking to instead, and the current style of docs have proven very hard for me to maintain (not for technical reasons, I just get paralysed trying to figure out the best way to introduce and order the content, and when making changes it means rewriting things all over the place rather than just being able to update the relevant section). 

I've also found the new doc writing process to be helpful as it's really putting a spotlight on the parts that are weird or tricky to explain, making me consider the necessity of the way things are just now. 🙂

**Comment by JordanMartinez** [comment]
>  Jordan and Thomas have done a great job with providing additional documentation material that I'll be linking to instead, and the current style of docs have proven very hard for me to maintain (not for technical reasons, I just get paralysed trying to figure out the best way to introduce and order the content, and when making changes it means rewriting things all over the place rather than just being able to update the relevant section).

So... I should continue to maintain `learn-halogen` and not drop it now that 'official' docs are being made for v5? Good to know!

**Comment by garyb** [comment]
@JordanMartinez yes please, if you're happy to continue doing so!

**Comment by JordanMartinez** [comment]
Ok. I'll continue to maintain it. Even so, I'm realizing that it only teaches the fundamentals of how Halogen works. it doesn't really explain how to use Halogen to solve problems. I think any lessons at that level would probably be better served/maintained as some paid online course rather than a free documentation project
